test/python/quantum_info/operators/symplectic/test_sparse_pauli_op.py

```diff
@@ -140,6 +140,14 @@ def test_sparse_pauli_op_init(self):
             coeffs[:] = 0
             self.assertEqual(spp_op, ref_op)
 
+    def test_sparse_pauli_op_init_long_ys(self):
+        """Test heavy-weight SparsePauliOp initialization."""
+        yyy = SparsePauliOp("Y" * 100)
+        print(yyy.coeffs)
+        yy = SparsePauliOp("Y" * 99)
+        print(yy.coeffs)
```

Comment:
you should remove the "print" statements from the test

Reply:
addressed